### PR TITLE
feat: add fucntion which change hexademical number to decimal

### PR DESCRIPTION
**Pull request by arim-kim**
Add fucntion which change hexadecimal number to decimal. And add css style file.
when hexa decimal is inputed, and submit button clicked, user can get decimal number.

### CODE REVIEW

File: views/index.ejs [hunk 1]
```diff
@@ -23,6 +23,11 @@
                 <button class ="input_button" type="submit">submit</button>
            </form>
            <p>   &nbsp decimal = <%= octalTodeci %></p>
+            <form action="/inputHexa" method="POST">
+                <input class ="input_box" type="text" name = "hexa" placeholder=" 8진수">
```
Comment (kche1106) [inline views/index.ejs]: You have to change "8진수" to "16진수".

**Review by kche1106 — CHANGES_REQUESTED** [review]
You have to change "8진수" to "16진수".

File: views/index.ejs (hunk 1 above)
Comment (arim-kim) [reply views/index.ejs]: ok. I got it. I will change.

**Review by kche1106 — APPROVED** [review]
Well done!

**Pull request merged**